src/resources/PlatformResources.ts

```diff
@@ -26,59 +27,61 @@ import UsageAnalytics from './UsageAnalytics/UsageAnalytics';
 import User from './Users/User';
 
 const resourcesMap: Array<{key: string; resource: typeof Resource}> = [
```

Comment:
its only a reorder ?

Reply:
yes, alphabetically - it was annoying me